lib/config/folderconfiguration.go

```diff
@@ -62,11 +62,11 @@ func (f FolderConfiguration) ModTimeWindow() time.Duration {
 		if usage, err := disk.Usage(f.Filesystem().URI()); err != nil {
 			dur = 2 * time.Second
 			l.Debugf(`Detecting FS at "%v" on android: Setting mtime window to 2s: err == "%v"`, f.Path, err)
-		} else if usage.Fstype == "" || strings.Contains(strings.ToLower(usage.Fstype), "fat") || strings.Contains(strings.ToLower(usage.Fstype), "msdos") {
+		} else if strings.Contains(strings.ToLower(usage.Fstype), "ext2") || strings.Contains(strings.ToLower(usage.Fstype), "ext3") || strings.Contains(strings.ToLower(usage.Fstype), "ext4") {
```

Comment:
How about flipping it around to:
`if strings.Contains("ext2ext3ext4", strings.ToLower(usage.Fstype)) {`
?

Reply:
```
if strings.HasPrefix(strings.ToLower(usage.Fstype), "ext") {
```

Reply:
> How about flipping it around to:
> `if strings.Contains("ext2ext3ext4", strings.ToLower(usage.Fstype)) {`
> ?

This won't work, will it? What we're actually looking for is likely `ext2/ext3` and/or possibly `ext2/ext3/ext4`.

> `if strings.HasPrefix(strings.ToLower(usage.Fstype), "ext") {`

This probably would be fine, but I personally think it's still safer to look for the values specifically. Who knows what Android will report there in the future 😕. Another possibility could be go regex and use something like `^ext[2-4]`, but I'm not sure if there's any real benefit over the current method.